ui/app/configuration-set-record/_record.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+(function () {
+  /* global _ */
```

Comment:
Please don't use this JSHint override here. For library functions you can add globals to the project in the .jshintrc  file; there's a globals object you can add it to, it's better practice.